index.html

```diff
@@ -111,8 +111,8 @@ <h2 class="about_primary_text">About me</h2>
         </div>
         <div id="contactMe" class="about_bottom flex-column">
             <div class="text_box">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
-            <form action="https://formspree.io/f/xeqwroeb" class="about_form" method="post">
-                <ul class="flex-column">
+            <form action="https://formspree.io/f/xeqwroeb" class="contactForm" method="post">
+                <ul class="flex-column contactFormList">
                     <li>
                         <label for="name">Name</label>
                         <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
```

Comment:
[OPTIONAL] This is optional ❓ as this was not the purpose of this review but consider adding more padding inside of the inputs, the placeholder text shouldn't be touching the border 👇🏽 

![Screenshot 2023-04-14 at 11 12 51](https://user-images.githubusercontent.com/45739963/232083996-77afe0bd-1cd3-4e69-9963-c6914c1cac00.png)